rrfs-test/testinput/rrfs_fv3jedi_hyb_2022052619.yaml

```diff
@@ -109,7 +109,7 @@ cost function:
          obsdataout:
            engine:
              type: H5File
-             obsfile: rass_tsen_obs_2022052619_hofx.nc4
+             obsfile: jdiag_rass_tsen_obs_2022052619.nc4
```

Comment:
Copy lines 101-112 here:
```
  observations:
     observers:
     - obs space:
         name: Aircraft
         obsdatain:
           engine:
             type: H5File
             obsfile: Data/obs/rass_tsen_obs_2022052619.nc4
         obsdataout:
           engine:
             type: H5File
             obsfile: jdiag_rass_tsen_obs_2022052619.nc4
```

RASS is the ground-based "Radio Acoustic Sounding System". Just curious why Line 104 sets the obs_space.name as "Aircraft"?

Reply:
@guoqing-noaa, probably just a mistake. I think we should try to clean up those obs space names to match the obs. There were a few other places where multiple obs spaces had the same name in the same yaml. Will changing those names affect ctest results. I would think not, but I'm not sure how the ctest references work and if they use the obs space names or not.

Reply:
The fv3-jedi yamls were restructured in #238 to use the same (or very close) observations to mpas-jedi, so we shouldn't need to worry about this particular bug anymore. But the new fv3-jedi yamls will need to have the corresponding changes made (renaming `hofxs` to `jdiag` and removing the `ombg` and `oman` diagnostics) as part of this PR. Sorry @guoqing-noaa!

Reply:
Thanks, @delippi and @SamuelDegelia-NOAA !

Yes, this issue is not related to this PR. But since I noticed this, I think I can bring this up for awareness. 
Thanks for addressing this issue in #238! No worries, I can resolve the conflicts once #238 is merged.